replayer/tests/functional_tests/binary_scripts/allocate_buffer.c

```diff
@@ -0,0 +1,13 @@
+#include <signal.h>
+#include <stdio.h>
+
+
+int main() {
+  char buffer[100];
+  printf("%lu\n", (unsigned long)buffer);
```

Comment:
Why do you print it's address?

Reply:
It's for the test, look at it tomorrow

Reply:
(This is a test script btw, you need to understand it in context of the tests)